templates/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
+
```

Comment:
Similar to above, you can likely avoid this file all together, we only have it in CAPA to support the e2e tests we are doing. The namespace substitution should be handled automatically when applied with `kubectl`

Reply:
I'm planning on adding k8s version customization after this so I'll keep it there